dockerfiles/build.sh

```
@@ -0,0 +1,52 @@
+    echo "Missing Dockerfile argument."
+    echo "Usage: ./dockerfiles/build.sh [/path/to/Dockerfile]"
+    exit 1
+fi
+
+__src_dir="$(dirname "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )")"
+
+__dockerfile="${1}"
+
+__context_dir=$(git rev-parse --show-toplevel 2>/dev/null)
+
+__repo_user=$(basename $(dirname $(git remote get-url origin | sed -E -e "s|[@:]|/|g")))
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)